integration-test/1910-underground-water.py

```diff
@@ -0,0 +1,86 @@
+# -*- encoding: utf-8 -*-
+from shapely.wkt import loads as wkt_loads
+import dsl
+from . import FixtureTest
+
+
+class UndergroundWaterTest(FixtureTest):
+    def test_water_level_1(self):
+        z, x, y = (16, 33199, 22547)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/way/236735251
+            dsl.way(236735251, dsl.tile_diagonal(z, x, y), {
```

Comment:
This OSM feature is a polygon, so we should be generating a `dsl.tile_box` polygon (not the tile diagonal line) for a better match to the true world, true data please.

Reply:
ah ok - I was just using the features generated from `mktest.py`, but this makes more sense